requirements.txt

```diff
@@ -1,10 +1,5 @@
-# This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
-
-py==1.4.14
-pytest==2.3.5
-pytest-xdist==1.8
-unittestzero
-requests==2.4.3
+py==1.4.30
```

Comment:
I know we do this elsewhere, but I'd like to let the version of pytest we use determine the version of py that it requires. In my opinion we can remove this line entirely.